prereise/gather/hiflddata/data_trans.py

```diff
@@ -0,0 +1,645 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+
+import csv
+import json
+import os.path
+import zipfile
+from collections import defaultdict
+
+import networkx as nx
+import pandas as pd
+from haversine import Unit, haversine
+
+Max_Value = 3000
+Min_Value = 0
+coord_precision = ".9f"
+default_base_kv = -999999.0
+
+
+def get_Zone(Z_csv):
+    """Generate a dictionary of zone using the zone.csv
+
+    :param str Z_csv: path of the zone.csv file
+    :return: (*dict*) -- a dict mapping the STATE to its ID.
+    """
+
+    zone = pd.read_csv(Z_csv)
+
+    # Create dictionary to store the mapping of states and codes
+    zone_dic = {}
+    for i in range(len(zone)):
+        zone_dic[zone["STATE"][i]] = zone["ID"][i]
```

Comment:
Is this equivalent to `zone_dic = dict(zip(zone.STATE, zone.ID))`?

Reply:
will change it in next revision.